app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -178,6 +177,11 @@
   }
 }
 
-.gem-c-feedback__submit {
-  @include button;
+// static.css on GOV.UK overwrites the component styles using input[type="text"]
+// so we need to apply  govuk-input styles using a stronger selector
+.gem-c-feedback input[type="text"] {
```

Comment:
It might be good to fix this in the input component itself rather than here, but I guess we're working towards killing off static.css anyway so maybe not worth the effort.

Reply:
Thanks @andysellick! Good point about having it in input. For the moment this is the only place I've seen the input being with static, I'll scout more potentially move it if necessary.